src/extras/PMREMGenerator.js

```diff
@@ -228,7 +228,7 @@ class PMREMGenerator {
 
 		} else { // Equirectangular
 
-			this._setSize( texture.image.width / 4 ?? 256 );
```

Comment:
@elalish `texture.image.width / 4` never gets `null` or `undefined` so `?? 256` is not needed here.

What use case did you want to catch here in the first place?

Reply:
This comes back to my API request to have textures have their own dimensions. Lots of textures have no `.image`, particularly those created from `WebGLRenderTarget`s. Most of the hacking in here is to get around the difficulty of trying to find their dimensions reliably. I sincerely hope there's a better way than what I've found.

Reply:
Didn't https://github.com/mrdoob/three.js/pull/23433 already solve this?

Reply:
Hopefully? I was just saying why I needed it in the first place. Also, I think this was for equirects rather than cube textures, which I don't think were addressed by that PR. I seem to recall having trouble with DataTextures as well.

Reply:
We can look into that. In the meantime... are you okay with @Mugen87's change?

Reply:
For sure; just be aware that testing is needed and I wouldn't be surprised to find more problems with texture dimensions not being found.

Reply:
`texture.image` should always be defined correctly for render targets and normal textures. There was a problem in context of cube textures but that should be fixed via #23433.